analyses/01_ebp-assembly-workflow/nextflow.config

```diff
@@ -1,6 +1,32 @@
+// Include NBIS profiles
+includeConfig '../../envs/nextflow/nbis_profiles.config'
+profiles {
+    // Override nac profile for EBP processes
+    nac {
+        includeConfig '../../envs/nextflow/nac.config'
+        params {
+            // NAC does not have enough memory to have the db in ram. (Slows FCSGX down a lot.)
+            fcs.ramdisk_path = ''
+        }
+        process {
+            // Increase resourceLimits to use highest memory nodes on NAC
+            withName: 'FCSGX_RUNGX' {
+                cpus       = 16
+                memory     = 370.GB
+                time       = 30.d
+                resourceLimits = [
+                    cpus: 16,
+                    memory: 370.GB,
+                    time: 30.d
+                ]
+            }
+        }
+    }
+}
+
```

Comment:
So let me see if I understood, the nac profile has set up resources and all for all processes but for 'FCSGX_RUNGX' we override the the nac profile to give it more memory.
shouldn't `fcs.ramdisk_path = ''` always be in the nac.config has it will always be a problem? or you would like that anything out of the ordinary goes in the `nbis_profiles.config` so then we never touch the` nac.config `in case nac changes?

Reply:
The idea is that the nac.config should not need changing. Anything workflow specific goes in the workflow specific directory `nextflow.config`. That's why the `fcs.ramdisk_path` is in the assembly config.